doc/classes/TreeItem.xml

```diff
@@ -492,6 +499,15 @@
 				Sets custom font size used to draw text in the given [param column].
 			</description>
 		</method>
+		<method name="set_edit_multiline">
+			<return type="void" />
+			<param index="0" name="column" type="int" />
+			<param index="1" name="multiline" type="bool" />
+			<description>
+				If [code]true[/code], the [param column] is multiline editable.
```

Comment:
```suggestion
				If [param multiline] is [code]true[/code], the [param column] is multiline editable.
```

Reply:
Done.

Reply:
Appreciate you updating others as well 👍